src/content/add-ons/overview.md

```diff
@@ -1,19 +1,28 @@
-**_Add-ons_ are components built specifically for IBM Cloud products and experiences. Carbon supports two main libraries, [Vanilla.js](https://github.com/ibm/carbon-components) and [React](https://github.com/ibm/carbon-components-react). There is also an unofficial [Angular library](https://pages.github.ibm.com/adaniel/angular-carbon-components/) for Carbon Components that is managed by an external team.**
+**_Add-on repositories_, or repos, are components built specifically for IBM Cloud products and experiences. The repos are built on top of the core Carbon repo, and enable teams to create their own custom components that follow Carbon's visual style and guidelines.**
```

Comment:
> built specifically for IBM Cloud products and experiences

Should we update the language here to be related to all of IBM?

Reply:
Yes, this should also be addressed across the entire website and documentation.